src/explorer/editors/FileEditor.ts

```diff
@@ -17,7 +16,7 @@ export class FileEditor extends BaseEditor<IAzureNode<FileTreeItem>> {
     }
 
     async getSaveConfirmationText(node: IAzureNode<FileTreeItem>): Promise<string> {
-        return `Saving '${node.treeItem.label}' will update the file "${node.treeItem.label}" in "${node.treeItem.site.name}".`;
+        return `Saving '${node.treeItem.label}' will update the file "${node.treeItem.label}" in "${node.treeItem.siteWrapper.name}".`;

```

Comment:
Please do a find/replace of 'siteWrapper.name' to 'siteWrapper.appName' before merging

(You only updated one reference in this file, but there are a few more and they should all be updated)